src/GovernorCountingFractional.sol

```diff
@@ -122,7 +138,8 @@ abstract contract GovernorCountingFractional is Governor {
     }
 
     /**
-     * @dev Count votes with full weight
+     * @dev Record votes with full weight cast for `support`. Reverts if partial
```

Comment:
Might expound a bit more here: "Because this function votes with the full weight of the delegate, it can only be called by a delegate once per proposal, and cannot be combined with a fractional vote before or after."

Reply:
Nice, done in d301ac2